tests/isosurfaces_simple_box_3D.prm

```diff
@@ -0,0 +1,62 @@
+# This input file tests the isosurface mesh refinment plugin
+include $ASPECT_SOURCE_DIR/tests/isosurfaces_simple_box_2D.prm
+
+set Dimension = 3
+
+# We take a box geometry and increase the repetitions
```

Comment:
These comments do not apply, please rm. Repetitions can also be removed, default is 1

Reply:
Still open. The comment about the repetitions does not apply and repetitions of 1 can be removed. I know it doesn't matter for the function of this test, it's just cleaner.